docs/user_guides/fs/feature_group/advanced_data_validation.md

```diff
@@ -0,0 +1,276 @@
+
+The introduction to data vaildation guide can be found [here](data_validation.md). The notebook example to get started with Data Validation in Hopsworks can be found [here](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/integrations/great_expectations/fraud_batch_data_validation.ipynb).
+
+## Data Validation Configuration Options in Hopsworks
+
+### Validation Ingestion Policy
+
+Depending on your use case you can setup data validation as a monitoring or gatekeeping tool when trying to insert new data in your Feature Group. Switch behaviour by using the `validation_ingestion_policy` kwarg:
+
+- `"ALWAYS"` is the default option and will attempt to insert the data regardless of the validation result. Hassle free, it is ideal to monitor data ingestion in a development setup.
+- `"STRICT"` is the best option for production ready projects. This will prevent insertion of DataFrames which do not pass all data quality requirements. Ideal to avoid "garbage-in, garbage-out" scenarios, at the price of a potential loss of data. Check out the best practice section for more on that.
+
+#### In the UI
+
+Go to the Feature Group edit page, in the Expectation section you can choose between the options above.
+
+#### In the python client
+
+```python3
+fg.expectation_suite.validation_ingestion_policy = "ALWAYS" # "STRICT"
+```
+
+If your suite is registered with Hopsworks, it will persist the change to the server.
+
+### Disable Data Validation
+
+Should you wish to do so, you can disable data validation on a punctual basis or until further notice.
+
+#### In the UI
+
+You can do it in the UI in the Expectation section of the Feature Group edit page. Simply tick or untick the enabled checkbox.
+
+#### In the python client
+
+```python3
+fg.expectation_suite.run_validation = False
+```
+
+If your suite is registered with Hopsworks, it will persist the change to the server.
+
+### Edit Expectations
+
+The one constant in life is change. If you need to add, remove or edit an expectation you can do it both in the UI or via the python client. Note that changing the expectation type or its corresponding feature will throw an error in order to preserve a meaningful validation history.
+
+#### In Hopworks UI
+
+Go to the Feature Group edit page, in the expectation section. You can click on the expectation you want to edit and edit the json configuration. Check out Great Expectations documentation if you need more information on a particular expectation.
+
+### In Hopsworks Python Client
+
+There are several way to edit an Expectation in the python client. You can use Great Expectations API or directly go through Hopsworks. In the latter case, if you want to edit or remove an expectation, you will need the Hopsworks expectation ID. It can be found in the UI or in the meta field of an expectation. Note that the Feature Group and corresponding Expectation Suite need to be registered to enable the Expectation API:
```

Comment:
Could we have a link sending back to the registration documentation or something like that for "Note that the Feature Group and corresponding Expectation Suite need to be registered to enable the Expectation API"

Reply:
Reworded to be clear by what I meant with registered.